### PR TITLE
Use correct scheme in w3c-test.org link

### PR DESCRIPTION
**Pull request by zcorpan**
Tests should use http scheme by default and https scheme if the path has ".https.".

(This is rebased on current master. Please "squash and merge".)

### CODE REVIEW

File: webapp/components/interop.html [hunk 1]
```diff
@@ -324,6 +328,10 @@ <h2>Interoperability</h2>
       stateFunc(state, '', `/interop${path}?q=${encodedQuery}`);
     }
 
+    computeTestScheme(path) {
+      return path.indexOf('.https.') !== -1 ? 'https' : 'http';
```
Comment (foolip) [inline webapp/components/interop.html]: A link to the logic in wpt run which this needs to be in sync with would be nice. I'd be surprised if there are any mismatches even if the rule isn't exactly the same, so not asking to centralize the definition or anything :)

Comment (zcorpan) [reply webapp/components/interop.html]: Added.

**Comment by lukebjerring** [comment]
Deployed to https://web-platform-tests-zcorpan-https-dot-here-lift-this.appspot.com

**Comment by zcorpan** [comment]
Seems to do what it should there :)

https://web-platform-tests-zcorpan-https-dot-wptdashboard.appspot.com/WebCryptoAPI/idlharness.https.html links to https
https://web-platform-tests-zcorpan-https-dot-wptdashboard.appspot.com/2dcontext/building-paths/canvas_complexshapes_arcto_001.htm links to http

**Pull request merged**